packages/csv-parser-price/src/cli.js

```diff
@@ -85,14 +87,12 @@ Convert commercetools price CSV data to JSON.`,
   })
 
   .option('logLevel', {
-    default: 'info',
     describe: 'Logging level: error, warn, info or verbose.',
   })
-  .coerce('logLevel', (arg) => {
-    npmlog.level = arg
-  })
   .argv
 
+npmlog.level = args.logLevel || minLogLevel
```

Comment:
This will set a `logLevel` to error when the output is `stdout` unless user set it to anything else.

So if you want to print a result to stdout, only errors are displayed but you still have an option to turn on more verbose logging if you want to debug it.